tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
Oh, no, totally missed those :man_facepalming: just saw the `length` tests in `tests/unit/array` and foolishly thought there weren't any for `sort`